projects/Mercier Vanderlinden Asset Management Portfolio/info.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "Mercier Vanderlinden Asset Management Portfolio",
+    "description": "The Portfolio app of Mercier Vanderlinden lets customers easily, always and everywhere access the composition, performance and history of their portfolios.",
+    "publisher": "Mercier Vanderlinden Asset Management",
```

Comment:
Add Mercier Vanderlinden Asset Management as a Company in the Company folder as well.

Reply:
done